codecs/rotate/package.json

```diff
@@ -2,6 +2,10 @@
   "name": "rotate",
   "scripts": {
     "build:image": "docker build -t squoosh-rotate .",
-    "build": "docker run --rm -v $(pwd):/src squoosh-rotate ./build.sh"
+    "build": "docker run --rm -v $(pwd):/src squoosh-rotate ./build.sh",
+    "benchmark": "echo File size after gzip && npm run benchmark:filesize && echo Baseline && npm run benchmark:baseline && echo Optimizing && npm run benchmark:optimizing",
+    "benchmark:baseline": "v8 --liftoff --no-wasm-tier-up --no-opt ./benchmark.js",
+    "benchmark:optimizing": "v8 ./benchmark.js",
```

Comment:
@bmeurer @mathiasbynens Would you say these are good flags to test the baseline wasm compiler vs the optimizing one?

Reply:
I don't know TBH. @hammacher and/or @gahaas might now.

Reply:
In order to ensure that you never run baseline wasm code, you should add "--no-liftoff --no-wasm-tier-up" to the optimizing config.